fresque/lib/views.py

```diff
@@ -73,10 +78,17 @@ def newpackage(db, method, data, username):
             db.commit()
         except SQLAlchemyError:
             result.flash.append(("An error occurred while adding your "
-                 "package, please contact an administrator.", "danger"))
+                "package, please contact an administrator.", "danger"))
         else:
             result.flash.append(("Package successfully created!", "success"))
             result.redirect = ('package', {"name": pkg.name})
+
+            # Add git repo creation message
+            if create_git_repo(pkg.name, gitfolder):
```

Comment:
Hi @ralphbean @abompard, I have made required changes as you have suggested. Sorry for the delay as I was traveling.